pkgs/development/python-modules/simplekv/default.nix

```diff
@@ -1,4 +1,9 @@
 {
+  azure-storage-blob,
+  boto3,
+  google-cloud-storage,
+  pymongo,
+  redis,
```

Comment:
Non-blocking nit: Group these dependencies together at the bottom with `dulwich`

Reply:
My preference (and what I thought I had already done) would be to fully alphabetize the input attrset by name. Is this some particular nixpkgs pattern? Or would alphabetization be okay?

Reply:
I'll just put this here :)
```nix
{
  lib,
  buildPythonPackage,
  fetchFromGitHub,
  pythonOlder,
  setuptools,

  # optional dependencies
  azure-storage-blob,
  boto3,
  dulwich,
  google-cloud-storage,
  pymongo,
  redis,

  # testing
  mock,
  pytestCheckHook,
  six,
}:
```

As previously stated, it's a nonblocking nit, so ofc I would also accept your PR if you find this idea to be silly ^^

Reply:
I honestly don't care enough to continue the discussion. I'll apply this and move on. :)